pkpdapp/pkpdapp/models/variable.py

```diff
@@ -13,7 +13,29 @@
 from django.db.models import Q
 
 
-class Variable(models.Model):
+class BaseVariable(models.Model):
```

Comment:
I think you should put everything from Variable in `BaseVariable`, except for the link to the model (e.g. you will need a unit). So you've got access to all of that in your StoredVariable as well